scripts/apps/authoring-react/authoring-react.tsx

```diff
@@ -1429,15 +1416,13 @@ export class AuthoringReact<T extends IBaseRestApiResponse> extends React.PureCo
                     <WithInteractiveArticleActionsPanel location="authoring">
                         {() => (
                             <Layout.AuthoringFrame
-                                header={primaryToolbarWidgets.length < 1 &&
-                                    this.props.getAuthoringPrimaryToolbarWidgets == null ? null : (
+                                header={primaryToolbarWidgets.length < 1
+                                    && extraPrimaryToolbarWidgets?.length < 1 ? null : (
                                         <SubNav>
                                             <AuthoringToolbar
                                                 entity={state.itemWithChanges}
                                                 coreWidgets={primaryToolbarWidgets}
-                                                extraWidgets={
-                                                    this.props.getAuthoringPrimaryToolbarWidgets(exposed)
-                                                }
+                                                extraWidgets={extraPrimaryToolbarWidgets}
```

Comment:
This coreWidgets/extraWidgets thing isn't doing anything, but joins 2 arrays together. Can you simply accept "widgets" as a prop and do the joining of lists before passing?